cd-scripts/cdRelease.py

```diff
@@ -55,6 +76,19 @@ def create_and_push_tag(repo, tag):
         sys.exit(1)
 
 
+def parse_pr(opts):
+    resp = requests.get(
+        f"https://api.github.com/repos/sky-uk/mite/pulls/{opts['--pr_number']}"
+    )
+    pr_message = resp.json()["body"]
+
+    matches = re.findall(r"- \[x\] (\w+)(?:$|[\n\r])", pr_message, re.IGNORECASE)
```

Comment:
```suggestion
    matches = re.findall(r"^- \[x\] (major|minor|patch)(?:$|[\n\r])", pr_message, re.IGNORECASE)
```

Reply:
(the `^` at the beginning might need to be `^|\n` or something like that.  i forget off the top of my head how matching in multiline strings works...)

Reply:
I added `re.MULTILINE` to the flags, which makes it easier, so it's now:

```python
    matches = re.findall(
        r"^- \[x\] (major|minor|patch)\r$", pr_message, re.IGNORECASE | re.MULTILINE
    )
```

It's a bit annoying that github uses DOS line endings (`\r\n`) instead of the standard, so you have to do `\r$`, but it still looks better.